lazyflow/operators/classifierOperators.py

```diff
@@ -497,36 +478,45 @@ def propagateDirty(self, slot, subindex, roi):
         elif slot == self.PredictionMask:
             self.PMaps.setDirty(roi.start, roi.stop)
```

Comment:
for simplicity I'd opt to set everything dirty here, too

```suggestion
            self.PMaps.setDirty()
```

Reply:
done